code-challenges.test.js

```diff
@@ -12,22 +12,84 @@
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
-// a) Create a test with expect statements for each of the variables provided.
-
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// ***********************************************************************************************************************
+/*
+INPUT: NUMBER > 2 && there are two variables, so there should be two expect statements
+OUTPUT: and array that length is the size its input containing the numbers of the fibinachi sequence (n1,n2, n1+n2, n3+n2, n4+n3...)
+METHODS: using methods that alther number data types.
+there is gonna need to be the formula: fib[i] = fib[i-1] + fib[i-2];  where n>1
+*/
+// ***************************************************************************************************************************
+
+
+// a) Create a ***test*** with expect statements for each of the variables provided.
+/* SKELOTON: 
+Describe( `My Function` , ( parameter ) => {
```

Comment:
I would not use backticks instead of single or double quotes for your strings. Only for when you want to do string interpolation